sleap/io/cameras.py

```diff
@@ -1,21 +1,58 @@
 """Module for storing information for camera groups."""
+import logging
+from pathlib import Path
+import tempfile
+import cattr
+import toml
+from typing import List, Optional, Union, Iterator, Any, Dict, Tuple
 
-from typing import List, Optional, Union, Iterator
-
-from attrs import define, field
 from aniposelib.cameras import Camera, FisheyeCamera, CameraGroup
+from attrs import define, field
+from attrs.validators import deep_iterable, instance_of
 import numpy as np
 
+from sleap.util import deep_iterable_converter
+from sleap.io.video import Video
+
+
+logger = logging.getLogger(__name__)
+
 
 @define
 class Camcorder:
     """Wrapper for `Camera` and `FishEyeCamera` classes.
 
     Attributes:
         camera: `Camera` or `FishEyeCamera` object.
+        videos: List of `Video` objects.
     """
 
-    camera: Optional[Union[Camera, FisheyeCamera]] = field(factory=None)
+    camera: Union[Camera, FisheyeCamera]
+    camera_cluster: "CameraCluster" = None
```

Comment:
This would be cleaner, but I the `Camera` and `Camcorder` class are a bit tightly correlated whereas `Camcorder` depends more on the `Camera` class and needs to be defined after `Camera`... meaning I can't use `Camcorder` in the definition of the `Camera` class (but I can use it in `__attrs_post_initi__`)